### DISCUSSION TITLE
Need Help With 500 Error- Solved

### DISCUSSION THREAD

**Discussion by skijohn877** [GerryDazoo/Slinger#586] [General]
Hi All,
Recently purchased a 500, did a factory reset, found the ip and port. Looks like slinger starts ok and the password was reset to admin. However when I try to connect is saying another user. Tried rebooting the slingbox numerous times.

Any ideas?

Version : 4.01 Running on Linux-5.10.103-v7+-armv7l-with-debian-10.13 pid= 2311 slingbox_server.py
Using config file config.ini
Connection Manager Running on port 8880 with 2 max streams using URL slingbox.Broadcast Responder Running

BOXES [('sb1', 'SBGR')]
BOX URL /slingbox/SBGR
Building page for SBGR
Reading Custom Remote definition from remote.txt
starting up on  port 8880 8388608
Streamer Running:  2 config.ini SBGR SBGR 8880 8388608
slingbox Checking for slingbox at ('192.168.1.19', 5101) 0
slingbox ('192.168.1.19', 5101) OK
slingbox Using slingbox at  ('192.168.1.19', 5101)
Streamer:  slingbox Waiting for first stream, flushing any IR requests that arrive while not connected to slingbox
04/20/2023, 13:16:15.542   Streaming request from ('192.168.1.26', 60620)  /slingbox/SBGR 
STREAM=192.168.1.26:60620:0

Slinginfo  "350/500/M1" 12 30 192.168.1.19 5101 3072 10 admin
slingbox Starting Stream for  192.168.1.26:60620
slingbox Opening Control stream 0x2000 0 0
Connecting... ('192.168.1.19', 5101) Control
cmd: 0x7e err: 0x18 176
slingbox Box in use! Kicking off other user.
cmd: 0x7e err: 0x18 176
slingbox Box in use! Kicking off other user.
cmd: 0x7e err: 0x18 176
slingbox Box in use! Kicking off other user.
cmd: 0x7e err: 0x18 176
slingbox Cannot kick off other user, not starting session.
If this error persists, consider rebooting your slingbox
slingbox ERROR: Slingbox session startup failed.
slingbox Logging Out
Streamer:  slingbox Waiting for first stream, flushing any IR requests that arrive while not connected to slingbox

**Comment by kdurkee-cohere** [comment]
It seems you may not be defining your slingbox 'admin password' properly that you should have retrieved. Mine looks like this

`Info   sbtype           res fr   boxip boxport  encrypted info(password maybe?)`
Slinginfo  "350/500/M1/M2"   16 60 10.1.1.82  5201   3072 3  IAtijdioT6

**Comment by skijohn877** [comment]
So it took a few factory resets, but now it's working. Apparently, I needed to go through most of the setup, i.e. network selection, source, etc. It eventually craps out looking for equipment lists off the network. At that point I let it sit. Ran a port map against the IP and saw port 5201 open. 

Only one thing I discovered I have Fios TV One STB's, I can connect the Slingbox 500 with RCA cables, but not the HDMI. No component output. Apparently the new STB's are using HDCP. So looking for a work around for that issue.

**Reply by sp222222** [reply]
easy. get a HDCP splitter. I use one my setup.
